src/harness/cbsd_sas_interface.py

```diff
@@ -0,0 +1,167 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
```

Comment:
tell git you move sas_interface.py -> cbsd_sas_interface.py

BTW, this file includes not only cbsd_sas interface, but test data interface. Prefer not to rename.